### PR TITLE
Allow to hide version

### PR DESCRIPTION
**Pull request by akondas**
This PR add option `--hide-version` which allow to hide dependency version in generated file:
```
php-legal-licenses generate --hide-version
```

### CODE REVIEW

File: bin/php-legal-licenses [hunk 1]
```diff
@@ -9,7 +9,7 @@ if (file_exists(__DIR__.'/../vendor/autoload.php')) {
     require __DIR__.'/vendor/autoload.php';
 }
 
-$app = new Symfony\Component\Console\Application('PHP Legal Licenses', '1.1.0');
+$app = new Symfony\Component\Console\Application('PHP Legal Licenses', '1.2.0');
```
Comment (akondas) [inline bin/php-legal-licenses]: ready to release :ship:

**Comment by marmichalski** [comment]
Hi, what's the status of this PR? Could anyone review it, @orediggerco, @JohnRiv?

**Comment by orediggerco** [comment]
Hey @marmichalski, reviewing your PR now!

**Comment by orediggerco** [comment]
Merged and added version v1.1.1. Thanks for your contribution, it's appreciated!

**Comment by akondas** [comment]
Thanks.

**Pull request merged**